exercises/I-array-properties/exercise.js

```diff
@@ -6,7 +6,7 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  return arr.length <= 0; // complete this statement
```

Comment:
array.length should be >= 0. So it seems more logical to test arr.length === 0 ?